src/main.rs

```diff
@@ -1395,7 +1396,12 @@ async fn config_reset(config: Config, _args: &ConfigReset) -> Result<String, Err
 
 async fn tz_reset(config: Config, _args: &ConfigSetTimezone) -> Result<String, Error> {
     match config.set_timezone().await {
-        Ok(_) => Ok("Timezone set successfully.".to_string()),
+        Ok(updated_config) => {
+            let tz = updated_config
+                .timezone
+                .unwrap_or_else(|| "unknown".to_string());
```

Comment:
Could we make this string "None"

Reply:
Thx, is this what you meant?

Reply:
Wasn't clear if you meant "None" enum or "none" text.

Reply:
I think I misunderstood the code here, I'm happy with your solution 👍